core/rfb.js

```diff
@@ -64,6 +64,7 @@ export default class RFB extends EventTargetMixin {
         this._rfb_max_version = 3.8;
         this._rfb_tightvnc = false;
         this._rfb_xvp_ver = 0;
+        this._rfb_ultravnc_scinfo = '';
```

Comment:
I don't see any need to keep this around, so let it be local for the auth function.

Reply:
Ok, I will change it in the next commit.